operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1098,6 +1098,7 @@ ingester:
     #
     # storageClass: "-"
 
+  livenessProbe: {}
```

Comment:
Upon testing, it has turned out the `livenessProbe: {}` is invalid. The ingester container fails with

```
spec.containers[0].livenessProbe: Required value: must specify a handler type
```

I've updated the changes here, to use the default `null` for documenting